src/views/partials/searchbar.hbs

```diff
@@ -1,6 +1,9 @@
-<div id="search">
-  <form class="search__form" action="/articles" method="POST">
-     <input class="search__input" type="text" name="search" placeholder="Search by keyword">
-     <input type="submit" value="Submit">
+<div id="search" class="w-100 tc">
+  <form class="mb4 " action="/articles" method="POST">
+     <input class="input__reset br4 pa2" type="text" name="search" placeholder="Search by keyword">
+     <input class="input__reset bn
+            shadow-1 fw6 white br3 ml2
+            pa2 pl3 pr3 bg-light-blue" type="submit"
```

Comment:
Again, It would improve usability to add a hover effect wherein the cursor changes to a pointer when I hover over this button. (It doesn't happen automatically because this is an input element that we've styled to look like a button)